pycam/workspace/data_models.py

```diff
@@ -920,8 +920,8 @@ def _get_scaled_model(self, model):
             for key, current_size, target_size in zip(
                     ("scale_x", "scale_y", "scale_z"), model.get_dimensions(), axes):
                 if current_size == 0:
```

Comment:
I would guess, that I just messed up this conditional: it should have been `target_size == 0` instead of `current_size == 0` (since the old error message indicated just this).

Or what do you think?

(feel free to squash your commits into one, if you like)

Reply:
Yes, checking target size would make more sense. But if  `kwargs[key] = target_size / current_size`, cases of `current_size == 0` should still be treated separately (ZeroDivisionError).
So I think it should be:
- if target_size == 0: raise exception
- elif target_size == None: don't scale
- elif current_size == 0: don't scale
- else: scale by (target_size / current_size)

And I didn't know about commit squashing, thank you

Reply:
Your summary sounds reasonable to me.

Just one minor detail: `target_size == None` -> `target_size is None` (but you probably just wanted to show the idea instead of the precise comparisons).

I would be happy to merge such changes!